products/jquery-ui.md

```diff
@@ -0,0 +1,97 @@
+---
+title: jQuery UI
+category: framework
+tags: javascript-runtime
+permalink: /jquery-ui
+changelogTemplate: https://github.com/jquery/jquery-ui/releases/tag/__LATEST__
+releaseDateColumn: true
+
+identifiers:
+-   purl: pkg:github/jquery/jquery-ui
+-   purl: pkg:maven/org.webjars/jquery-ui
+-   purl: pkg:npm/jquery-ui
+-   purl: pkg:nuget/jQuery.UI.Combined
+
+auto:
+  methods:
+  -   git: https://github.com/jquery/jquery-ui.git
+
+releases:
+-   releaseCycle: "1.14"
+    releaseDate: 2024-08-05
+    eol: false
+    latest: "1.14.0"
+    link: https://blog.jqueryui.com/2024/08/jquery-ui-1-14-0-released/
+
+-   releaseCycle: "1.13"
+    releaseDate: 2021-10-07
+    eol: true
```

Comment:
Assuming these went EOL once the next release cycle, we should be able to put dates (for this and the remaining cycles).

```suggestion
    eol: 2024-08-05
```